src/Services/ModerationLogService.cs

```diff
@@ -62,9 +62,14 @@ private Embed CreateEmbedResponse(ModerationLogEntry logEntry)
         {
             EmbedBuilder builder = new EmbedBuilder();
 
-            StringBuilder author = new StringBuilder($"[{logEntry.ActionType.Humanize()}]");
             StringBuilder description = new StringBuilder();
+            StringBuilder author;
```

Comment:
You aren't really building anything, I'd suggest you simplify it to a multi-line ternary operator:
```cs
string author = logEntry.InfractionId > -1
    ? $"[{logEntry.InfractionId}] [{logEntry.ActionType.Humanize()}]"
    : $"[{logEntry.ActionType.Humanize()}]";
```

Reply:
You don't see it here but author is appended on line 76 and 78. It was StringBuilder before, I just duplicated it.